src/platform/packages/shared/kbn-es/src/stateful_resources/roles.yml

```diff
@@ -129,3 +129,71 @@ system_indices_superuser:
       resources: ['*']
   run_as: ['*']
 
+# temporarily added to move forward with Scout
+platform_engineer:
```

Comment:
I hope we can merge https://github.com/elastic/kibana/pull/213798 tomorrow and avoid updating default definitions.